Hangman Game/README.md

```diff
@@ -0,0 +1,6 @@
+# The task is to create a hangman game.
```

Comment:
Readme should strictly follow [this](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md) template.